### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by ajank**
Just two words corrected.

### CODE REVIEW

**Review by akshayparopkari — APPROVED** [review]

**Comment by akshayparopkari** [comment]
Thank you, @ajank 
Looks good to me 👍

I'll wait for one more approval before merging this PR.

**Review by esallychang — APPROVED** [review]
All looks good to me, thanks!

**Comment by esallychang** [comment]
Since we were just waiting for my approval, I am going ahead and merging this pull request, thanks @ajank !

**Pull request merged**
